src/pyfaf/queries.py

```diff
@@ -1027,6 +1027,12 @@ def get_unassigned_reports(db, report_type, min_count=0):
         query = query.filter(st.Report.count >= min_count)
     return query.all()
 
+def unassign_reports(db, report_ids):
```

Comment:
Can you add a description of this function, so I know what it does without looking at the code?

Reply:
Mmm… okay. Though it is quite obvious in conjunction with the method name.

Reply:
Added a docstring.